### ISSUE TITLE
How to replace the activation function SiLU with ReLU

### ISSUE THREAD

**Issue by qingqinghu1026** [ultralytics/ultralytics#6014]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

How to replace the activation function SiLU with ReLU?
I tried to modify the following areas, but retraining did not take effect.
`class BottleneckCSP(nn.Module):
    """CSP Bottleneck https://github.com/WongKinYiu/CrossStagePartialNetworks."""

    def __init__(self, c1, c2, n=1, shortcut=True, g=1, e=0.5):  # ch_in, ch_out, number, shortcut, groups, expansion
        super().__init__()
        c_ = int(c2 * e)  # hidden channels
        self.cv1 = Conv(c1, c_, 1, 1)
        self.cv2 = nn.Conv2d(c1, c_, 1, 1, bias=False)
        self.cv3 = nn.Conv2d(c_, c_, 1, 1, bias=False)
        self.cv4 = Conv(2 * c_, c2, 1, 1)
        self.bn = nn.BatchNorm2d(2 * c_)  # applied to cat(cv2, cv3)
        # self.act = nn.SiLU()
        self.act = nn.ReLU()
        self.m = nn.Sequential(*(Bottleneck(c_, c_, shortcut, g, e=1.0) for _ in range(n)))

    def forward(self, x):
        """Applies a CSP bottleneck with 3 convolutions."""
        y1 = self.cv3(self.m(self.cv1(x)))
        y2 = self.cv2(x)
        return self.cv4(self.act(self.bn(torch.cat((y1, y2), 1))))`
`class Conv(nn.Module):
    """Standard convolution with args(ch_in, ch_out, kernel, stride, padding, groups, dilation, activation)."""
    #default_act = nn.SiLU()  # default activation
    default_act = nn.ReLU()

    def __init__(self, c1, c2, k=1, s=1, p=None, g=1, d=1, act=True):
        """Initialize Conv layer with given arguments including activation."""
        super().__init__()
        self.conv = nn.Conv2d(c1, c2, k, s, autopad(k, p, d), groups=g, dilation=d, bias=False)
        self.bn = nn.BatchNorm2d(c2)
        self.act = self.default_act if act is True else act if isinstance(act, nn.Module) else nn.Identity()

    def forward(self, x):
        """Apply convolution, batch normalization and activation to input tensor."""
        return self.act(self.bn(self.conv(x)))

    def forward_fuse(self, x):
        """Perform transposed convolution of 2D data."""
        return self.act(self.conv(x))`

**Comment by glenn-jocher** [comment]
@qingqinghu1026 you can use `act: nn.ReLU()` in a model YAML and then load the YAML. If you want to load pretrained weights you can do that after, see https://docs.ultralytics.com/modes/train/#usage-examples

**Comment by qingqinghu1026** [comment]
> @qingqinghu1026 you can use `act: nn.ReLU()` in a model YAML and then load the YAML. If you want to load pretrained weights you can do that after, see https://docs.ultralytics.com/modes/train/#usage-examples

Thanks for your answer,I solve it.

**Comment by glenn-jocher** [comment]
That's great to hear, @qingqinghu1026! I'm glad you were able to solve it. If you have any other questions or face any other issues, don't hesitate to ask. Happy coding with YOLOv8!

**Comment by 123123del** [comment]
Now using act: nn.ReLU() or activation: nn.ReLU()?

**Comment by Paul1229** [comment]
Could you provide an example of a YAML configuration file setting ReLU? Thank you very much.

**Comment by Y-T-G** [comment]
You can change this

https://github.com/ultralytics/ultralytics/blob/8d203cf40dadb9343241d2f65bd4c84820f27430/ultralytics/nn/modules/conv.py#L39
